src/app/ngrx/reducers/selectors/export.selectors.ts

```diff
@@ -22,85 +46,110 @@ import { selectRoutedGroupCharted } from './group-charted.selectors';
+        if (chartedPodcastDownloads && podcastDownloads.charted && chartedEpisodeDownloads && chartedEpisodeDownloads.length) {
+          exportData = [chartedPodcastDownloads, ...chartedEpisodeDownloads];
+        } else if (
+          chartedPodcastDownloads &&
+          podcastDownloads.charted &&
+          chartedPodcastDownloads.data.length &&
+          !(chartedEpisodeDownloads && chartedEpisodeDownloads.length)
+        ) {
+          exportData = [chartedPodcastDownloads];
+        } else if (chartedEpisodeDownloads && chartedEpisodeDownloads.length) {
+          exportData = chartedEpisodeDownloads;
+        }
+        break;
+      case CHARTTYPE_PODCAST:
+        if (chartedPodcastDownloads && chartedPodcastDownloads.data.length) {
+          exportData = [chartedPodcastDownloads];
+        }
+        break;
+      case CHARTTYPE_EPISODES:
+        if (chartedEpisodeDownloads && chartedEpisodeDownloads.length) {
+          exportData = chartedEpisodeDownloads;
+        }
+        break;
+    }
+    return exportData;
   }
-  return exportData;
-});
+);
 
 export const selectExportDropday = createSelector(
   selectRouter,
   selectSelectedEpisodeDropdays,
   (routerParams, dropdays: EpisodeDropday[]): ExportData[] => {
-    return dropdays && dropdays.filter(dropday => dropday.downloads).map(dropday => {
-      switch (routerParams.chartType) {
-        case CHARTTYPE_HORIZBAR:
-          return {
-            label: dropday.title,
-            guid: dropday.guid,
-            publishedAt: dropday.publishedAt,
-            total: getTotal(dropday.downloads)
-          };
-        case CHARTTYPE_EPISODES:
-          return {
-            label: dropday.title,
-            guid: dropday.guid,
-            publishedAt: dropday.publishedAt,
-            data: cumDownloads(dropday.downloads)
-          };
-      }
-    });
+    return (
+      dropdays &&
+      dropdays
+        .filter(dropday => dropday.downloads)
+        .map(dropday => {
+          switch (routerParams.chartType) {
+            case CHARTTYPE_HORIZBAR:
+              return {
+                label: dropday.title,
+                guid: dropday.guid,
+                publishedAt: dropday.publishedAt,
+                total: getTotal(dropday.downloads)
+              };
+            case CHARTTYPE_EPISODES:
+              return {
+                label: dropday.title,
+                guid: dropday.guid,
+                publishedAt: dropday.publishedAt,
+                data: cumDownloads(dropday.downloads)
+              };
+          }
+        })
+    );
   }
 );
 
-export const toCsvArray = (downloads: ExportData[], dateFormat?: Function): string[][] => {
+export const selectExportListeners = createSelector(selectRoutedPodcastListeners, (routedPodcastData: PodcastListeners) => {
+  if (routedPodcastData && routedPodcastData.listeners) {
+    return [
+      {
+        label: 'Unique Listeners',
+        data: routedPodcastData.listeners
```

Comment:
I find it a bit weird that the dates are the x-axis, and all the data ends up on row 2.  But I guess that's the way our other chart exports work (where there are multiple lines).